app/models/ror_org.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
I'd make the case for this being an inline class in RorService:
* That is the only place it is used.
* It makes the context for this class clear. (In `models/`, it has no context as to why it exists and what it is used for.)
* It is not an active record model.

(If inline, you can ignore my comments below.)

Reply:
Inlined Org class with RoRService and also added a couple comments